realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -83,15 +83,32 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_nativeCreate(
 }
 
 JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_createNativeWithImplicitTransactions
-  (JNIEnv* env, jobject, jlong native_replication_ptr, jbyteArray keyArray)
+  (JNIEnv* env, jobject, jlong native_replication_ptr, jint durability, jbyteArray keyArray)
```

Comment:
createNativeWithImplicitTransactions and createNative shares a lot of code. They should be refactored so we don't duplicate that code.